src/stores/authContext.tsx

```diff
@@ -34,6 +35,7 @@ export const AuthProvider = ({ children }: { children: React.ReactNode }) => {
   const [user, setUser] = useState<SessionUser | null>(null)
   const [portalUser, setPortalUser] = useState<PortalUser | null>(null)
   const router = useRouter()
+  const { setUserIdFn, unsetUserIdFn } = useAnalytics()
```

Comment:
Question: what do you think about mocking these out so we can add a test to ensure they are called in `authContext.test.tsx`? Is that something you tried? Maybe the value of that is low?

Reply:
@gidjin I have some tests for `analyticsContext` that are calling each of these functions and verifying that by looking at console output. I'm not sure how I would go about mocking that out for the `authContext` to verify the same. As for the level of value that would bring, I'm not sure? The code coverage report is showing that the functions are reached in `authContext`, and the tests in `analyticsContext` are checking the same events that would be sent to Matomo when the login/logout functions run. The combination of those two seem fine to me, but curious to know if others agree?

Reply:
@jcbcapps I added some expectations to the authContext. If they look good to you I'm good with merging this in.